azimuth_caas_operator/models/v1alpha1/cluster.py

```diff
@@ -44,7 +44,6 @@ class ClusterSpec(schema.BaseModel):
     # partially described by the cluster type ui-meta
     extraVars: schema.Dict[str, schema.Any] = pydantic.Field(default_factory=dict)
 
-
 class Cluster(
```

Comment:
I think that extra space is required by the formatter, which is probably causing one of the test failures you have.